src/components/application_manager/test/commands/mobile/reset_global_properties_test.cc

```diff
@@ -263,35 +331,52 @@ TEST_F(ResetGlobalPropertiesRequestTest,
       hmi_apis::Common_Result::eType::UNSUPPORTED_RESOURCE;
 
   (*message_)[am::strings::msg_params][am::strings::properties][0] =
-      mobile_apis::GlobalProperty::VRHELPTITLE;
+      mobile_apis::GlobalProperty::TIMEOUTPROMPT;
+  (*message_)[am::strings::msg_params][am::strings::properties][1] =
+      mobile_apis::GlobalProperty::MENUICON;
 
   ON_CALL(mock_app_manager_, application(kConnectionKey))
       .WillByDefault(Return(app_));
 
-  EXPECT_CALL(*app_, reset_vr_help_title());
-  EXPECT_CALL(*app_, reset_vr_help());
+  EXPECT_CALL(mock_app_manager_, get_settings())
+      .WillOnce(ReturnRef(mock_app_manager_settings_));
 
-  EXPECT_CALL(*app_, set_reset_global_properties_active(true));
+  std::vector<std::string> time_out_prompt;
+  time_out_prompt.push_back("time_out");
+  EXPECT_CALL(mock_app_manager_settings_, time_out_promt())
+      .WillOnce(ReturnRef(time_out_prompt));
 
-  smart_objects::SmartObjectSPtr vr_help =
-      ::utils::MakeShared<smart_objects::SmartObject>(
-          smart_objects::SmartType_Map);
-  EXPECT_CALL((*am::MockMessageHelper::message_helper_mock()),
-              CreateAppVrHelp(_)).WillOnce(Return(vr_help));
+  EXPECT_CALL(*app_, set_timeout_prompt(_));
+
+  smart_objects::SmartObjectSPtr prompt =
+      utils::MakeShared<smart_objects::SmartObject>();
+  (*prompt) = "prompt";
+  smart_objects::SmartObject* ptr = prompt.get();
+  EXPECT_CALL(*app_, timeout_prompt()).WillOnce(Return(ptr));
```

Comment:
@okozlovlux code looks same as upper but variable name and value are prompt except timeout prompt.